other/limit-configmap-for-sa/limit_configmap_for_sa.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: limit-configmap-for-sa
+  annotations:
+    policies.kyverno.io/title: Limit ConfigMap for a User
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: ConfigMap
+    policies.kyverno.io/description: Limits given ConfigMap for a specific users.
```

Comment:
This policy shows how to restrict certain operations on specific ConfigMaps by ServiceAccounts.